### PR TITLE
feat(dict): Add more corrections

### PR DESCRIPTION
**Pull request by Flakebi**
I encountered these when going through a codebase with another tool.

### CODE REVIEW

File: crates/typos-dict/assets/words.csv [hunk 1]
```diff
@@ -12035,7 +12051,6 @@ coralina,carolina
 corasir,corsair
 coratia,croatia
 corcodile,crocodile
-cordinate,coordinate
```
Comment (epage) [inline crates/typos-dict/assets/words.csv:12038]: How come this correction was removed?

File: crates/typos-dict/assets/words.csv [hunk 2]
```diff
@@ -29561,6 +29598,8 @@ maraudeurs,marauder
 marbels,marbles
 marbleds,marbles
 marchmallows,marshmallows
+marcos,macros
```
Comment (epage) [inline crates/typos-dict/assets/words.csv:29601]: Trying to decided what to do about this one because Marcos is a common enough of a name

**Comment by epage** [comment]
Wow, thanks for all of this!

File: crates/typos-dict/assets/words.csv (hunk 1 above)
Comment (Flakebi) [reply crates/typos-dict/assets/words.csv:12038]: Ups, that was unintentional

File: crates/typos-dict/assets/words.csv (hunk 2 above)
Comment (Flakebi) [reply crates/typos-dict/assets/words.csv:29601]: Fair point, I’ll remove it

**Comment by epage** [comment]
I'll go ahead and wait on the other PR before releasing, if thats ok with you

**Comment by Flakebi** [comment]
Sure, no hurry. Thanks for taking a look so fast!

**Pull request merged**